app/experimenter/targeting/constants.py

```diff
@@ -763,17 +788,6 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-DOESNT_PREFER_REDUCED_MOTION = NimbusTargetingConfig(
-    name="Users Who Don't Prefer Reduced Motion",
-    slug="users_who_dont_prefer_reduced_motion",
-    description="Users who don't prefer reduced motion",
-    targeting="!userPrefersReducedMotion",
-    desktop_telemetry="",
-    sticky_required=False,
-    is_first_run_required=False,
-    application_choice_names=(Application.DESKTOP.name,),
-)
```

Comment:
Oh let's leave this in here.  We'd need a more detailed plan for deprecating an existing targeting config.

Reply:
I just added this a few days ago with [issue 7959](https://github.com/mozilla/experimenter/issues/7959), and it isn't in use as far as I'm aware. I'm happy to leave it in if you think it will be of use though!

Reply:
Oh! Okay then yeah, should be fine.  Thanks!